frontend/src/features/admin-form/create/builder-and-design/MagicFormBuilder/components/MagicFormBuilderPromptModal.tsx

```diff
@@ -0,0 +1,198 @@
+    prompt:
+      'registrations for an event, including sections for personal details, event details and acknowledgement to terms and conditions.',
+  },
+  {
+    label: 'Grant applications',
+    prompt:
+      'applications for government grants from business entities, incorporating sections for project details, budget breakdown and applicant qualifications.',
+  },
+  {
+    label: 'Feedback collection',
+    prompt:
+      'feedback including rating scales for service quality satisfaction, justification for rating, specific feedback on what is good and what is bad and potential suggestions for improvement.',
+  },
+]
+
+const PromptSelectorBar = ({
+  promptIdeas,
+  onClick,
+}: {
+  promptIdeas: {
+    label: string
+    prompt: string
+  }[]
+  onClick: (prompt: string) => void
+}) => {
+  return (
+    <Flex direction="column">
+      <Text textStyle="subhead-1">Need inspiration? Try one of these:</Text>
+      <HStack
+        display="flex"
+        justifyContent="space-between"
+        overflowX="auto"
+        gap="0.25rem"
+        mt="0.75rem"
+      >
+        {promptIdeas.map((idea) => (
+          <Button
+            key={idea.label}
+            variant="clear"
+            size="xs"
+            borderRadius="4px"
+            backgroundColor="primary.100"
+            _focus={{
+              backgroundColor: 'primary.200',
+              textColor: 'primary.600',
+            }}
+            _hover={{
+              backgroundColor: 'primary.200',
+            }}
+            onClick={() => {
+              onClick(idea.prompt)
+            }}
+          >
+            {idea.label}
+          </Button>
+        ))}
+      </HStack>
+    </Flex>
+  )
+}
+
+const GENERATE_FORM_PLACEHOLDER =
+  'Describe your form, including fields and sections to create'
+
+export interface TextPromptInputs {
+  prompt: string
+}
+
+const MagicFormBuilderCreateFormPrompt = ({
+  onSubmit,
+  isSubmitLoading,
+  onCancel,
+}: {
+  onSubmit: (textPromptInputs: string) => void
+  isSubmitLoading: boolean
+  onCancel: () => void
+}) => {
+  const {
+    register,
+    handleSubmit,
+    setValue,
+    formState: { errors },
+  } = useForm<TextPromptInputs>()
+
+  return (
+    <>
+      <ModalHeader display="flex" alignItems="center">
+        <Text textStyle="h2">Create fields with AI</Text>
+        <Badge
+          colorScheme="primary"
+          variant="subtle"
+          color="secondary.500"
+          ml="0.5rem"
+        >
+          Beta
+        </Badge>
+      </ModalHeader>
+      <ModalBody>
+        <FormControl isRequired isInvalid={!!errors.prompt?.message}>
+          <FormLabel textStyle="subhead-1">
+            I want to create a form that collects...
+          </FormLabel>
+          <Textarea
+            minH="9rem"
+            borderRadius="4px"
+            placeholder={GENERATE_FORM_PLACEHOLDER}
+            {...register('prompt', {
+              required: 'Please enter a prompt.',
```

Comment:
We don't usually have a fullstop on this iirc.
```suggestion
              required: 'Please enter a prompt',
```